cargo-marker/Cargo.toml

```diff
@@ -34,3 +34,6 @@ tracing            = { workspace = true }
 tracing-error      = { workspace = true }
 tracing-subscriber = { workspace = true, features = ["env-filter"] }
 yansi              = { workspace = true }
+
+[dev-dependencies]
+expect-test = "1.4"
```

Comment:
Let's put this version requirement in the `[workspace,dependencies]` in the root-level `Cargo.toml`